pkg/hubble/exporter/config_watcher.go

```diff
@@ -67,7 +67,7 @@ func (c *configWatcher) reload() {
 	config, hash, err := c.readConfig()
 	if err != nil {
 		DynamicExporterReconfigurations.WithLabelValues("failure").Inc()
-		c.logger.Warnf("failed reading dynamic exporter config")
+		c.logger.Warnf("failed reading dynamic exporter config: %s", err)
```

Comment:
Let's adopt proper structured logging patterns when adding an error to the message.

```suggestion
		c.logger.WithError(err).Warn("failed reading dynamic exporter config")
```